src/routes/(console)/project-[region]-[project]/overview/platforms/createAndroid.svelte

```diff
@@ -83,14 +83,18 @@ const val APPWRITE_PUBLIC_ENDPOINT = "${sdk.forProject(page.params.region, page.
     }
 
     onMount(() => {
-        const unsubscribe = realtime.forConsole(page.params.region, 'console', (response) => {
-            if (response.events.includes(`projects.${projectId}.ping`)) {
-                connectionSuccessful = true;
-                invalidate(Dependencies.ORGANIZATION);
-                invalidate(Dependencies.PROJECT);
-                unsubscribe();
+        const unsubscribe = realtime.forConsole(
+            page.params.region,
+            ['console', 'project'],
```

Comment:
This is main change, adding project to channel list. Becasue Console is only FRA region, while projects are per-region